library/src/scala/tasty/Tasty.scala

```diff
@@ -2,4 +2,4 @@ package scala.tasty
 
 import scala.tasty.reflect._
 
```

Comment:
Reformat the following line to be shorter?

Reply:
Done